generate-templates.yaml

```diff
@@ -215,7 +215,19 @@
 
   - name: Generate Windows templates
     template:
-      src: win2k12r2.tpl.yaml
+      src: windows.tpl.yaml
+      dest: "{{ playbook_dir }}/dist/templates/windows-{{ item.workload }}-{{ item.flavor }}.yaml"
+    with_items:
+    - {flavor: medium, workload: server, memsize: "4Gi", cpus: 1, iothreads: False, tablet: False}
```

Comment:
@ksimon1 are 4Gi enough to run the windows machine?

Reply:
According to osinfo-db, win 2019 have recommended RAM size: https://gitlab.com/libosinfo/osinfo-db/blob/master/data/os/microsoft.com/win-2k19.xml.in#L26

Reply:
Thanks, than I am OK with it.